dmd_D2Integral.m

```diff
@@ -0,0 +1,36 @@
+function D2Int=dmd_D2Integral(N,L,x,b,w)
```

Comment:
I would suggest implementing this short function into a subfunction of distanceDiracGaussian.m instead of in a separate file. This would keep it more readable.